Dockerfile.client

```diff
@@ -14,27 +14,26 @@ RUN --mount=type=cache,target=/root/.cache \
 ENV SOLANA_BIN_PATH="/root/.local/share/solana/install/active_release/bin"
 ENV PATH="$SOLANA_BIN_PATH:$PATH"
 
-ADD ethereum /usr/src/ethereum
 WORKDIR /usr/src/ethereum
-RUN --mount=type=cache,target=/root/.cache \
-	--mount=type=cache,target=/root/.npm \
-    npm ci
+COPY ethereum/package.json ethereum/package-lock.json ./
+RUN npm ci
+COPY ethereum ./
```

Comment:
We might want to keep the cache `--mount` to speed up `npm ci` runs whenever package-lock.json changes?